modules/installation-configuring-shiftstack-single-ipv6.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-shiftstack-single-ipv6_{context}"]
+= Configuring a cluster with single-stack IPv6 networking
+
+You can create a single-stack IPv6 cluster on {rh-openstack-first} after you configure your {rh-openstack} deployment.
+
+IMPORTANT: You cannot convert a dual-stack cluster into a single-stack IPv6 cluster. Additionally, deployments that rely on Fast Datapath are not supported.
```

Comment:
I was chatting with @EmilienM and we think it might not be best to mention the no fast data path support as there is some on going work.
@rlobillo WDYT?

Reply:
Sure, I'm ok with that